src/PlusImageProcessing/vtkPlusBoneEnhancer.cxx

```diff
@@ -168,6 +170,17 @@ PlusStatus vtkPlusBoneEnhancer::ReadConfiguration(vtkSmartPointer<vtkXMLDataElem
   if (imageProcessingOperations != NULL)
   {
 
+    // Read SaveIntermediateResults tag
+    vtkSmartPointer<vtkXMLDataElement> SaveIntermediateResultsBool = imageProcessingOperations->FindNestedElementWithName("SaveIntermediateResults");

```

Comment:
Local variables with lower case please: saveIntermediateResultsBool
The way we differentiate between member and local variables is the case of the first letter, so this is good to keep consistent to increase readability.